plugins/woocommerce/src/Internal/Admin/Loader.php

```diff
@@ -319,7 +329,13 @@ public static function add_component_settings( $settings ) {
 			);
 		}
 
+		/**
+		 * The woocommerce_component_settings_preload_endpoints filter
+		 *
+		 * @since 6.5.0
```

Comment:
Using 6.5.0 since that's the first version released after we merged WC Admin into the core.